Model/ExifToolWrapper.cs

```diff
@@ -41,27 +41,47 @@
 
 using System;
 using System.Collections.Generic;
-using System.Linq;
 using System.Text;
-using System.Threading.Tasks;
 using System.Diagnostics;
 using System.IO;
 using GeoTagNinja;
+using static System.Windows.Forms.VisualStyles.VisualStyleElement;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+using System.Collections;
 
 namespace ExifToolWrapper
```

Comment:
I think this file should be okay apart from having renamed a few things - I don't remember splitting this into anything further.